src/DialogflowEntityClient.js

```diff
@@ -0,0 +1,52 @@
+import { EntityTypesClient } from 'dialogflow';
+
+export class EntityNotFoundError extends Error { }
+
+export class DialogflowEntityClient {
+  constructor(entityTypeName, projectId) {
+    this.entityTypeName = entityTypeName;
+    this.projectId = projectId;
+  }
+
+  async initialize() {
```

Comment:
Could use lazy initialization with [this](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Functions/get).